src/shared/components/case-viewer/case-view/case-view.component.ts

```diff
@@ -31,7 +31,9 @@ export class CaseViewComponent implements OnInit {
           this.caseDetails = caseView;
           this.caseService.announceCase(caseView);
         })
-      ).toPromise();
+      )
+      .toPromise()
```

Comment:
Small refactor to make it consistent with other extracted components error handling.